src/components/PhotoCard/PhotoCard.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import * as classes from './PhotoCard.module.scss'
 
-const PhotoCard = ({ img, name, username, profileImg, style }) => 
+const PhotoCard = ({ img, name, profileImg, style }) => 
   <div className={classes.PhotoCardWrapper} style={style}>
     <div className={classes.ImgWrapper}>
       {img}
```

Comment:
What is your thinking behind not using username here below the name? Cleaner?

I included it because I think some people, be it a minority, might associate more with usernames than actual names.

Reply:
Yeah. I was thinking of it being cleaner and that their profile here would be an extension of their real life artist personal. So a username is just a means of finding them, not identifying them.